core/src/main/java/com/scalar/db/sql/TwoPhaseCommitTransactionSession.java

```diff
@@ -18,14 +18,13 @@
 import com.scalar.db.sql.exception.UnknownTransactionStatusException;
 import com.scalar.db.sql.statement.DdlStatement;
 import com.scalar.db.sql.statement.DmlStatement;
-import com.scalar.db.sql.statement.SelectStatement;
 import com.scalar.db.sql.statement.Statement;
 import java.util.Objects;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.NotThreadSafe;
 
 @NotThreadSafe
-public class TwoPhaseCommitTransactionSqlSession implements SqlSession {
+public class TwoPhaseCommitTransactionSession implements SqlStatementSession {
```

Comment:
Renamed `TwoPhaseCommitTransactionSqlSession` to `TwoPhaseCommitTransactionSession`.